message_forwarder.go

```diff
@@ -56,7 +57,7 @@ func extractUUID(msg string) (string, error) {
 	contextRegexp := regexp.MustCompile(uuidContextRegexp)
 	validRegexp := regexp.MustCompile(uuidValidRegexp)
 	uuidContext := contextRegexp.FindString(msg)
-	uuid := validRegexp.FindString(uuidContext)
+	uuid := strings.ToLower(validRegexp.FindString(uuidContext))
```

Comment:
Why is the forwarder validating our messages? I'd have thought the forwarder is "dumb" and just forwards messages for us. What if we decide we need to forward a message that doesn't contain a UUID? We won't be able to as it will fail the validation on the bridge.